src/styles/_calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  box-sizing: border-box;
+  padding: $padding;
+  width: ($padding * 2) + ($gap * 6) + ($size * 7);
+  display: flex;
+  align-items: center;
+  gap: $gap;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
```

Comment:
maybe this will be a better solution in this case?
this property will work for the entire project
```
* {
  box-sizing: border-box;
}
```

Reply:
the instructions recommend avoiding this selector so as not to decrease performance